extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/DataSourceBuildTimeConfig.java

```diff
@@ -15,11 +16,11 @@
     public Optional<String> driver;
 
     /**
-     * Whether we want to use XA.
+     * Whether we want to use regular JDBC transactions, XA, or disable all transactional capabilities.
      * <p>
-     * If used, the driver has to support it.
+     * When enabling XA you will need a driver implementing {@link javax.sql.XADataSource}.
      */
-    @ConfigItem(defaultValue = "false")
-    public boolean xa;
+    @ConfigItem(defaultValue = "ENABLED")
```

Comment:
This is very minor but for the best doc generation experience, it should be `enabled` lowercase (the accepted values are listed as lowercase with hyphens in the doc).

Reply:
thanks, didn't know that case didn't matter - expected it had to match the Enum name. weird.